samples/csharp_dotnetcore/48.customQABot-all-features/README.md

```diff
@@ -0,0 +1,175 @@
+
+### Configure knowledge base of the project
+- Follow instructions [here][Quickstart] to create a Custom question answering project. You will need this project's name to be used as `ProjectName` in [appsettings.json](appsettings.json).
+- Visit [Language Studio][LS] and open created project.
+- Go to `Edit knowledge base` -> Click on `...` -> Click on `Import questions and answers` -> Click on `Import as TSV`.
+- Import [SampleForCQA.tsv](CognitiveModels/SampleForCQA.tsv) file.
+- You can test your knowledge base by clicking on `Test` option.
+- Go to `Deploy knowledge base` and click on `Deploy`.
+
+### Connect your bot to the project.
+Follow these steps to update [appsettings.json](appsettings.json).
+- In the [Azure Portal][Azure], go to your resource.
+- Go to `Keys and Endpoint` under Resource Management.
+- Copy one of the keys as value of `LanguageEndpointKey` and Endpoint as value of `LanguageEndpointHostName` in [appsettings.json](appsettings.json).
+- `ProjectName` is the name of the project created in [Language Studio][LS].
+
+## To try this sample
+
+- Install the Bot Framework Emulator version 4.14.0 or greater from [here][BFE]
+- Clone the repository
+
+    ```bash
+    git clone https://github.com/Microsoft/botbuilder-samples.git
+    ```
+
+- In a terminal, navigate to `samples/csharp_dotnetcore/48.customQABot-all-features`
+- Run the bot from a terminal or from Visual Studio, choose option A or B.
+
+  A) From a terminal
+
+  ```bash
+  # run the bot
+  dotnet run
+  ```
+
+  B) Or from Visual Studio
+
+  - Launch Visual Studio
+  - File -> Open -> Project/Solution
+  - Navigate to `samples/csharp_dotnetcore/48.customQABot-all-features` folder
+  - Select `CustomQABotAllFeatures.csproj` file
+  - Press `F5` to run the project
+- Connect to the bot using Bot Framework Emulator
+  1) Launch Bot Framework Emulator
+  2) File -> Open Bot
+  3) Enter a Bot URL of `http://localhost:3978/api/messages`
+
+## Try Active Learning
+- Try the following utterances:
+  1) Surface Book
+  2) Power
+- In Language Studio, click on inspect to see the closeness in the scores of the returned answers.
+- In [Bot Framework Emulator][BFE], a card is generated with the suggestions.
+  - Clicking an option would send a [feedback record](https://docs.microsoft.com/en-us/rest/api/cognitiveservices/questionanswering/question-answering-projects/add-feedback) which would show as suggestion under `Review suggestions` in [Language Studio][LS].
+  - `ActiveLearningCardTitle`, `ActiveLearningCardNoMatchText` and `ActiveLearningCardNoMatchResponse` in the card could be changed from [RootDialog.cs](Dialogs/RootDialog.cs).
+
+## Try Multi-turn prompt
+- Try the following utterances:
+  1) Accessibility
+  2) Options
+- You will notice that multi-turn prompts associated with the question are also returned in the responses.
+
+## Try Precise Answering
+- Try the following utterances:
```

Comment:
See suggestions from samples/csharp_dotnetcore/12.customQABot/README.md for lines 79-86

Reply:
Same here, content seems fine when rendered in github:

![image](https://user-images.githubusercontent.com/12264946/165850669-cd6ab34b-0775-43c9-b14b-f61bace67231.png)